### PR TITLE
Fix coercion `TorQuadModMor` -> `AutomorphismGroupElem{TorQuadMod}`

### PR DESCRIPTION
**Pull request by StevellM**
@simonbrandhorst

### CODE REVIEW

File: src/Groups/abelian_aut.jl [hunk 1]
```diff
@@ -81,6 +81,7 @@ function (aut::AutGrpAbTor)(f::Union{GrpAbFinGenMap,TorQuadModMor};check::Bool=t
   gene = GAP.Globals.GeneratorsOfGroup(AA)
   img = GAP.Obj([img_gap(a) for a in gene])
   fgap = GAP.Globals.GroupHomomorphismByImagesNC(AA,AA,img)
+  @req fgap in aut.X "Map does not define an element of the group"
```
Comment (simonbrandhorst) [inline src/Groups/abelian_aut.jl]:
```suggestion
  check || fgap in aut.X || error("Map does not define an element of the group")
```

Comment (StevellM) [reply src/Groups/abelian_aut.jl]: In fact, thanks!

Comment (simonbrandhorst) [reply src/Groups/abelian_aut.jl]: Ah, I confused && and ||

Comment (thofma) [reply src/Groups/abelian_aut.jl]: `@req check && fgap in aut.X "..."`?

Comment (StevellM) [reply src/Groups/abelian_aut.jl]: Is it a preferable choice or a matter of taste ? Right now it is in line with the first input check of the function. Would you suggest to change both of them ?

**Review by simonbrandhorst — APPROVED** [review]

**Comment by simonbrandhorst** [comment]
It works.

File: src/Groups/abelian_aut.jl (hunk 1 above)
Comment (thofma) [reply src/Groups/abelian_aut.jl]: `@req` will throw an `ArgumentError` and not a generic `ErrorException`. It is used only for argument checking. I guess it is not really appropriate at in this line. Would be for the first line though.

**Pull request merged**